### ISSUE TITLE
dt.render.date should check for a null value

### ISSUE THREAD

**Issue by challgren** [ypnos-web/cakephp-datatables#66]
When using the provided dt.render.date if the date passed is null an error will occur.

**Comment by ypnos-web** [comment]
This is in line with the default renderer. What would you want it to do/render in case of missing date? Do you think the behavior should be configurable (e.g. a 'fallback text' argument)?

I am also open for a pull request in this matter.

**Comment by ypnos-web** [comment]
Yes, your observation is correct. What I don't understand is why you use custom parsing of the string in the first method.

Would you like to prepare a pull request with the fallback mechanics?
